content/1.5/docs/_index.md

```diff
@@ -3,10 +3,21 @@ title: "Documentation"
 date: 2024-02-20
 cascade:
   - toc_root: true
-    type: docs 
+    type: docs
 ---
 
 # Orchestrator
 
 Choose a section from the list below. For Orchestrator introduction, check the [Quick Start](./quickstart/).
 
+# Release notes
+
+We are pleased to announce following features delivered as part of 1.5 release
+
+* Improved user experience
```

Comment:
should links be added to the items?

Reply:
would you like to see figma linked? I am not sure whether it would be accessible to people viewing this